assignments/life_expectancy/tests/test_cleaning.py

```diff
@@ -1,17 +1,23 @@
 """Tests for the cleaning module"""
+from unittest.mock import patch
 import pandas as pd
-
-from life_expectancy.cleaning import clean_data
+from life_expectancy.loading_saving_data import save_data, load_data, file_path
+from life_expectancy.data_cleaning import clean_data
 from . import OUTPUT_DIR
 
 
-def test_clean_data(pt_life_expectancy_expected):
+@patch("life_expectancy.loading_saving_data.save_data")
+@patch("life_expectancy.loading_saving_data.load_data")
+def test_clean_data(
+    mock_load, mock_save, life_expectancy_original, pt_life_expectancy_expected
+):
     """Run the `clean_data` function and compare the output to the expected output"""
-    #clean_data()
-    clean_data(OUTPUT_DIR / "eu_life_expectancy_raw.tsv",'PT')
-    pt_life_expectancy_actual = pd.read_csv(
-        OUTPUT_DIR / "pt_life_expectancy.csv"
-    )
+    mock_load.return_value = life_expectancy_original
+    mock_save.side_effect = print()
```

Comment:
normally you use print when you _actually_ want to print something

Reply:
@margarida-borbinha given the requirements of the assignment, why don't you make a suggestion?

In general you should strike an appropriate balance between pointing out problems and providing direct guidance

Reply:
ok